test/e2e/clusterCSINodeV2_test.go

```diff
@@ -62,12 +68,13 @@ func TestClusterCSINodeV2(t *testing.T) {
 		}
 	})
 
-	namespacedName := types.NamespacedName{
-		Name:      testutil.TestClusterCRName,
-		Namespace: namespace,
-	}
 	t.Run("ClusterStatusCheck", func(t *testing.T) {
-		if err = testutil.ClusterStatusCheck(t, namespacedName, 1, testutil.RetryInterval, testutil.Timeout); err != nil {
+		if err = testutil.ClusterStatusCheck(t, clusterNsName, 1, testutil.RetryInterval, testutil.Timeout); err != nil {
+			logs, logErr := util.GetOperatorLogs(namespace)
+			if logErr != nil {
+				t.Error(errors.Wrap(logErr, "failed to fetch operator logs"))
+			}
+			t.Log(logs)
```

Comment:
I suggest to Log first and than check logErr, because for me looks more readable if Error and Fatal appear at the end

Reply:
I need to leave the Fatal at the end as it exits, but I'm re-arranged to try and make it more readable.

Reply:
perfect i just say:
```
t.Log(logs)
if logErr != nil t.Error(errors.Wrap(logErr, "failed to fetch operator logs"))
t.Fatal(err)
```
Because Error doesn't stops execution.